ui/app/rif/components/searchDomains.js

```diff
@@ -14,21 +14,35 @@ class SearchDomains extends Component {
     showDomainRegisterPage: PropTypes.func,
     showDomainsDetailPage: PropTypes.func,
     getDomainDetails: PropTypes.func,
+    getStoredDomains: PropTypes.func,
   }
 
+  componentDidMount () {
+    this.loadDomains();
+  }
+
+  loadDomains () {
+    this.props.getStoredDomains().then(domains => {
+      this.setState({
+        storedDomains: domains,
+      });
+    })
+    }
+
   _handleKeyDown = (e) => {
     if (e.key === 'Enter') {
       const typedDomain = cleanDomainName(e.target.value);
-      // There is a limitation in manager that domains with less 5 characters are blocked
-      if (typedDomain.length < 5) {
+      // There is a limitation in rns that domains with less 5 characters are blocked
+      if (typedDomain.length <= 5) {
```

Comment:
Please, delete the equals from here, typedDomain can be of 5 chars

Reply:
Oh ok this was a merge issue i guess.